src/app/components/Transcript/TranscriptTimestamp/index.styles.tsx

```diff
@@ -0,0 +1,22 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  time: ({ mq }: Theme) =>
+    css({
+      // paddingInlineEnd: `${spacings.DOUBLE}rem`,
+      float: 'inline-start',
+      // marginBottom: 'inherit',
+      width: '100%',
+      [mq.GROUP_1_MIN_WIDTH]: {
+        width: '25%',
```

Comment:
Don't think we need to vary the width perbreakpoint, could we just use width auto?

Reply:
Yep good spot.